share/pot/clone.sh

```diff
@@ -313,6 +324,33 @@ pot-clone()
 					${EXIT} 1
 				fi
 				;;
+			d)
+				case $OPTARG in
+					"inherit")
+						_dns="inherit"
+						;;
+					"pot")
+						_dns="pot"
+						;;
+					"off")
+						_dns="off"
+						;;
+					custom:*)
+						if [ -r "${OPTARG##custom:}" ]; then
+							_dns=$OPTARG
+						else
+							_error "The file ${OPTARG##custom:} is not valid or reabale"
+							clone-help
+							${EXIT} 1
+						fi
+						;;
+					*)
+							_error "The dns ${OPTARG} is not valid option"
```

Comment:
"The dns" sounds wrong to my (non-native) ears, I would suggest:

```suggestion
							_error "'${OPTARG}' is not valid dns option"
```